hack/update-generated-conversions.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
Can you explain how to use this at https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/devel/api_changes.md#edit-version-conversions?